specs/bug6Spec.js

```diff
@@ -3,7 +3,7 @@ var test = require('./test');
 describe('bug 6', function() {
     it('Set flex-shrink to 1 by default', function(done) {
         var input = 'div{flex: 1;}';
-        var output = 'div{flex: 1 1;}';
+        var output = 'div{flex-grow: 1;flex-shrink: 1;}';
```

Comment:
I think `flex: 1` should become `flex: 1 1 0%` (or `flex: 1 1 0`, I can't remember if and why unitless is better) because `flex: 1` also sets `flex-basis`, so this fix isn't correct.

Reply:
IE11 do not understand unitless :(